p4rt_app/scripts/p4rt_program_table.cc

```diff
@@ -153,7 +168,8 @@ int main(int argc, char** argv) {
   }
   std::vector<sai::Update> pd_entries = std::move(*pd_entries_or);
 
-  P4rtTableWriter writer(std::move(p4rt_session), FLAGS_cleanup);
+  P4rtTableWriter writer(std::move(p4rt_session), FLAGS_cleanup,
+                         FLAGS_cleanall);
```

Comment:
Have you done a build ?

Reply:
Yes, the build is success here.

Reply:
Build Results:
==============
divya@0f547eb1bd4b:/sonic/src/sonic-p4rt/sonic-pins$ bazel build $BAZEL_BUILD_OPTS p4rt_app/...
INFO: Analyzed 92 targets (3 packages loaded, 123 targets configured).
INFO: Found 92 targets...
INFO: Elapsed time: 120.977s, Critical Path: 27.79s
INFO: 238 processes: 137 internal, 101 linux-sandbox.
INFO: Build completed successfully, 238 total actions


divya@0f547eb1bd4b:/sonic/src/sonic-p4rt/sonic-pins$ bazel build $BAZEL_BUILD_OPTS p4_pdpi/...
INFO: Analyzed 70 targets (1 packages loaded, 90 targets configured).
INFO: Found 70 targets...
INFO: Elapsed time: 4.551s, Critical Path: 3.78s
INFO: 4 processes: 1 internal, 3 linux-sandbox.
INFO: Build completed successfully, 4 total actions

UT Results:
===========
divya@0f547eb1bd4b:/sonic/src/sonic-p4rt/sonic-pins$ bazel test $BAZEL_BUILD_OPTS p4rt_app/...
INFO: Analyzed 92 targets (0 packages loaded, 0 targets configured).
INFO: Found 61 targets and 31 test targets...
INFO: Elapsed time: 24.991s, Critical Path: 5.04s
INFO: 32 processes: 1 internal, 19 linux-sandbox, 12 local.
INFO: Build completed successfully, 32 total actions
//p4rt_app/event_monitoring:app_state_db_port_table_event_test PASSED in 0.5s
//p4rt_app/event_monitoring:config_db_node_cfg_table_event_test PASSED in 0.5s
//p4rt_app/event_monitoring:config_db_port_table_event_test PASSED in 0.5s
//p4rt_app/event_monitoring:state_verification_events_test PASSED in 0.5s
//p4rt_app/p4runtime:ir_translation_test PASSED in 0.4s
//p4rt_app/p4runtime:p4info_verification_schema_test PASSED in 0.4s
//p4rt_app/p4runtime:p4info_verification_test PASSED in 0.4s
//p4rt_app/p4runtime:packetio_helpers_test PASSED in 0.5s
//p4rt_app/sonic:app_db_acl_def_table_manager_test PASSED in 0.6s
//p4rt_app/sonic:app_db_manager_test PASSED in 0.4s
//p4rt_app/sonic:app_db_to_pdpi_ir_translator_test PASSED in 0.4s
//p4rt_app/sonic:packetio_impl_test PASSED in 0.3s
//p4rt_app/sonic:packetio_port_test PASSED in 0.3s
//p4rt_app/sonic:response_handler_test PASSED in 0.4s
//p4rt_app/sonic:state_verification_test PASSED in 0.2s
//p4rt_app/sonic/adapters:fake_sonic_db_table_test PASSED in 0.1s
//p4rt_app/tests:acl_table_test PASSED in 0.9s
//p4rt_app/tests:action_set_test PASSED in 0.6s
//p4rt_app/tests:api_access_test PASSED in 0.6s
//p4rt_app/tests:arbitration_test PASSED in 0.8s
//p4rt_app/tests:fixed_l3_tables_test PASSED in 2.1s
//p4rt_app/tests:forwarding_pipeline_config_test PASSED in 3.3s
//p4rt_app/tests:grpc_behavior_test PASSED in 5.0s
//p4rt_app/tests:p4_programs_test PASSED in 1.0s
//p4rt_app/tests:packetio_test PASSED in 3.2s
//p4rt_app/tests:port_name_and_id_test PASSED in 0.8s
//p4rt_app/tests:response_path_test PASSED in 1.5s
//p4rt_app/tests:role_test PASSED in 0.6s
//p4rt_app/tests/lib:app_db_entry_builder_test PASSED in 0.0s
//p4rt_app/utils:event_execution_time_monitor_test PASSED in 0.2s
//p4rt_app/utils:table_utility_test PASSED in 0.6s

Executed 31 out of 31 tests: 31 tests pass.
INFO: Build completed successfully, 32 total actions

divya@0f547eb1bd4b:/sonic/src/sonic-p4rt/sonic-pins$ bazel test $BAZEL_BUILD_OPTS p4_pdpi/...
INFO: Analyzed 70 targets (0 packages loaded, 0 targets configured).
INFO: Found 44 targets and 26 test targets...
INFO: Elapsed time: 2.758s, Critical Path: 0.38s
INFO: 27 processes: 41 linux-sandbox.
INFO: Build completed successfully, 27 total actions
//p4_pdpi:ir_tools_test PASSED in 0.4s
//p4_pdpi/netaddr:ipv4_address_and_network_address_test PASSED in 0.2s
//p4_pdpi/netaddr:ipv6_address_test PASSED in 0.2s
//p4_pdpi/netaddr:mac_address_test PASSED in 0.3s
//p4_pdpi/string_encodings:bit_string_test PASSED in 0.2s
//p4_pdpi/string_encodings:byte_string_test PASSED in 0.3s
//p4_pdpi/string_encodings:decimal_string_test PASSED in 0.0s
//p4_pdpi/string_encodings:decimal_string_test_runner PASSED in 0.0s
//p4_pdpi/string_encodings:hex_string_test PASSED in 0.0s
//p4_pdpi/string_encodings:hex_string_test_runner PASSED in 0.0s
//p4_pdpi/string_encodings:readable_byte_string_test PASSED in 0.2s
//p4_pdpi/testing:helper_function_test PASSED in 0.3s
//p4_pdpi/testing:info_test PASSED in 0.0s
//p4_pdpi/testing:info_test_runner PASSED in 0.1s
//p4_pdpi/testing:main_pd_test PASSED in 0.0s
//p4_pdpi/testing:packet_io_test PASSED in 0.0s
//p4_pdpi/testing:packet_io_test_runner PASSED in 0.1s
//p4_pdpi/testing:rpc_test PASSED in 0.1s
//p4_pdpi/testing:rpc_test_runner PASSED in 0.0s
//p4_pdpi/testing:sequencing_test PASSED in 0.1s
//p4_pdpi/testing:sequencing_test_runner PASSED in 0.0s
//p4_pdpi/testing:table_entry_gunit_test PASSED in 0.3s
//p4_pdpi/testing:table_entry_test PASSED in 0.0s
//p4_pdpi/testing:table_entry_test_runner PASSED in 0.1s
//p4_pdpi/utils:annotation_parser_test PASSED in 0.3s
//p4_pdpi/utils:ir_test PASSED in 0.3s

Executed 26 out of 26 tests: 26 tests pass.
INFO: Build completed successfully, 27 total actions